### ISSUE TITLE
Black and Blank Output

### ISSUE THREAD

**Issue by page29** [upscayl/upscayl#958]
### Checklist

- [X] I have **tried ALL** of the steps mentioned in the [troubleshooting guide](https://github.com/upscayl/upscayl/wiki/Troubleshooting).
- [X] I have **searched** for this in the [issues tab](https://github.com/upscayl/upscayl/issues?q=).

### Describe the Bug

Always get a blank/black image irrespective of the model used. No output file saved.  Searched prior threads & followed all the steps but no resolution.
![2](https://github.com/user-attachments/assets/db730a62-16fc-43d6-990c-562134de898b)

### To Reproduce

Select a photo, model and click on upscayl

### Upscayl Version (or commit hash)

2.11.5

### Platform

Linux

### OS Version

Windows 10 Enterprise 

### GPU Name

NVDIA RTX 2000 ADA Generation Laptop GPU

### Logs

```sh
⚙️ Getting saveImageAs from localStorage:  jpg
⚙️ Getting model from localStorage:  {"label":"General Photo (Remacri)","value":"remacri"}
⚙️ Getting gpuId from localStorage:  1
🔀 Setting model to 
🔀 Setting model to remacri
⚙️ Getting saveImageAs from localStorage:  jpg
⚙️ Getting model from localStorage:  {"label":"General Photo (Remacri)","value":"remacri"}
⚙️ Getting gpuId from localStorage:  1
⚙️ Getting saveImageAs from localStorage:  jpg
⚙️ Getting model from localStorage:  {"label":"General Photo (Remacri)","value":"remacri"}
⚙️ Getting gpuId from localStorage:  1
🔀 Setting model to 
🔀 Setting model to remacri
🔄 Resetting image paths
🎒 BACKEND REPORTED:  🖼️ Updating Image Path:  C:\Users\um063688\Desktop\dji.jpg
🎒 BACKEND REPORTED:  📄 Selected File Path:  C:\Users\um063688\Desktop\dji.jpg
🖼 Selected Image Path:  C:\Users\um063688\Desktop\dji.jpg
📁 Selected Image Directory:  C:\Users\um063688\Desktop
🖼 imagePath:  C:\Users\um063688\Desktop\dji.jpg
🔤 Extension:  jpg
🔄 Resetting Upscaled Image Path
🏁 UPSCAYL
🎒 BACKEND REPORTED:  ✅ Upscayl Variables:  {"model":"remacri","gpuId":"1","saveImageAs":"jpg","inputDir":"C:\\Users\\um063688\\Desktop","fileNameWithExt":"dji.jpg","outputDir":"C:\\Users\\um063688\\Desktop","outFile":"C:\\Users\\um063688\\Desktop\\dji_upscayl_4x_remacri.jpg","fileName":"dji","scale":"4","compression":"0","customWidth":"","useCustomWidth":false,"tileSize":null}
🎒 BACKEND REPORTED:  📢 Upscayl Command:  -i,C:\Users\um063688\Desktop\dji.jpg,-o,C:\Users\um063688\Desktop\dji_upscayl_4x_remacri.jpg,-m,C:\Program Files\Upscayl\resources\models,-n,remacri,-g,1,-f,jpg,-c,0
🎒 BACKEND REPORTED:  👶 Updating Child Processes:  {"binary":"C:\\Program Files\\Upscayl\\resources\\bin\\upscayl-bin","args":["C:\\Program Files\\Upscayl\\resources\\bin\\upscayl-bin","-i","C:\\Users\\um063688\\Desktop\\dji.jpg","-o","C:\\Users\\um063688\\Desktop\\dji_upscayl_4x_remacri.jpg","-m","C:\\Program Files\\Upscayl\\resources\\models","-n","remacri","-g","1","-f","jpg","-c","0"]}
🎒 BACKEND REPORTED:  🛑 Updating Stopped:  false
🎒 BACKEND REPORTED:  [0 Intel(R) RaptorLake-S Mobile Graphics Controller]  queueC=0[1]  queueG=0[1]  queueT=0[1]
[0 Intel(R) RaptorLake-S Mobile Graphics Controller]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[0 Intel(R) RaptorLake-S Mobile Graphics Controller]  fp16-p/s/a=1/1/1  int8-p/s/a=1/1/1
[0 Intel(R) RaptorLake-S Mobile Graphics Controller]  subgroup=32  basic=1  vote=1  ballot=1  shuffle=1

🚧 UPSCAYL_PROGRESS:  [0 Intel(R) RaptorLake-S Mobile Graphics Controller]  queueC=0[1]  queueG=0[1]  queueT=0[1]
[0 Intel(R) RaptorLake-S Mobile Graphics Controller]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[0 Intel(R) RaptorLake-S Mobile Graphics Controller]  fp16-p/s/a=1/1/1  int8-p/s/a=1/1/1
[0 Intel(R) RaptorLake-S Mobile Graphics Controller]  subgroup=32  basic=1  vote=1  ballot=1  shuffle=1

🎒 BACKEND REPORTED:  [1 NVIDIA RTX 2000 Ada Generation Laptop GPU]  queueC=2[8]  queueG=0[16]  queueT=1[2]
[1 NVIDIA RTX 2000 Ada Generation Laptop GPU]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[1 NVIDIA RTX 2000 Ada Generation Laptop GPU]  fp16-p/s/a=1/1/1  int8-p/s/a=1/1/1
[1 NVIDIA RTX 2000 Ada Generation Laptop GPU]  subgroup=32  basic=1  vote=1  ballot=1  shuffle=1

🚧 UPSCAYL_PROGRESS:  [1 NVIDIA RTX 2000 Ada Generation Laptop GPU]  queueC=2[8]  queueG=0[16]  queueT=1[2]
[1 NVIDIA RTX 2000 Ada Generation Laptop GPU]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[1 NVIDIA RTX 2000 Ada Generation Laptop GPU]  fp16-p/s/a=1/1/1  int8-p/s/a=1/1/1
[1 NVIDIA RTX 2000 Ada Generation Laptop GPU]  subgroup=32  basic=1  vote=1  ballot=1  shuffle=1

🎒 BACKEND REPORTED:  0.00%

🚧 UPSCAYL_PROGRESS:  0.00%


🎒 BACKEND REPORTED:  92.60%

🚧 UPSCAYL_PROGRESS:  92.60%

🎒 BACKEND REPORTED:  💯 Done upscaling
upscaledImagePath:  C:\Users\um063688\Desktop\dji_upscayl_4x_remacri.jpg
💯 UPSCAYL_DONE:  C:\Users\um063688\Desktop\dji_upscayl_4x_remacri.jpg
⚙️ Getting saveImageAs from localStorage:  jpg
⚙️ Getting model from localStorage:  {"label":"General Photo (Remacri)","value":"remacri"}
⚙️ Getting gpuId from localStorage:  1
```

**Comment by zawa999** [comment]
try this
https://bit.ly/4e4zLlq
select the desired compiler in the installer

**Comment by page29** [comment]
> try this https://bit.ly/4e4zLlq select the desired compiler in the installer

I am unable to open the bit.ly link. It says "_Warning: This URL has been blocked by Bitly's systems as potentially harmful._"

**Comment by page29** [comment]
I tried all those steps and I still get black image.

**Comment by Kongsea** [comment]
> I tried all those steps and I still get black image.

I have met the same problem...

**Comment by OIRNOIR** [comment]
I have this problem as well.

**Comment by page29** [comment]
Not sure why this is happening. It was working fine untill a few days ago and I was able to successfully upscale images. A few days ago I tried to upscale a 360 image which have a black output. Since then every photo gives a black output.

**Comment by aaronliu0130** [comment]
@Kongsea @OIRNOIR Please go through the checklist.

The logs are all normal, assuming you trimmed the middle. Was there a recent driver update? Are you sure Capsviewer reports capabilities.

**Comment by page29** [comment]
> @Kongsea @OIRNOIR Please go through the checklist.
> 
> The logs are all normal, assuming you trimmed the middle. Was there a recent driver update? Are you sure CapsViewer reports capabilities?

This is what I get when I run VulcanCapsViewer. Not sure which line item shows Vulcan Compatibality. 
![image](https://github.com/user-attachments/assets/4a5cc7a1-7dc1-47af-98de-9cb76557d7d7)

**Comment by OIRNOIR** [comment]
> I have this problem as well.

I managed to resolve my issue. It was failing the same as before, despite many graphics driver updates. I then switched my KDE plasma desktop to performance mode (system tray -> power management) and re-ran upscayl and it worked. It might have been a change in the way the balanced power mode (which I usually keep my laptop at) handles intensive GPU tasks.

Running Upscayl on intel integrated graphics isn't fun (or fast) but it works now, thankfully.

**Comment by Apertur3** [comment]
I've got the exact same issue and can't seem to solve it, on a MBP (M1 Max, 64gb, Sonoma 14.5).

**Comment by NayamAmarshe** [comment]
> I've got the exact same issue and can't seem to solve it, on a MBP (M1 Max, 64gb, Sonoma 14.5). <img alt="image" width="1663" src="https://private-user-images.githubusercontent.com/84971116/369626086-bac4638c-1d80-48ad-a42d-f776be977db7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY5NjQyMjgsIm5iZiI6MTcyNjk2MzkyOCwicGF0aCI6Ii84NDk3MTExNi8zNjk2MjYwODYtYmFjNDYzOGMtMWQ4MC00OGFkLWE0MmQtZjc3NmJlOTc3ZGI3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTIyVDAwMTIwOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ4ODk1MDE3YTY3MjU4ZTQ2OTEwZjQyZTJiZGZmMjYyZDQ1NTYzYjg0YzhjMWViZjE3YjQ4ZjhiMWQwODUxMDUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Tl4sM0zXmvPlTiqcjg36bkbLDAtwZAyD_ufcAsfYl20">
> 
> Log (I cut out most of the progress, but it did stop at 47.50%):

Yours is a different problem. #447

**Comment by Apertur3** [comment]
> > I've got the exact same issue and can't seem to solve it, on a MBP (M1 Max, 64gb, Sonoma 14.5). <img alt="image" width="1663" src="https://private-user-images.githubusercontent.com/84971116/369626086-bac4638c-1d80-48ad-a42d-f776be977db7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY5NjQyMjgsIm5iZiI6MTcyNjk2MzkyOCwicGF0aCI6Ii84NDk3MTExNi8zNjk2MjYwODYtYmFjNDYzOGMtMWQ4MC00OGFkLWE0MmQtZjc3NmJlOTc3ZGI3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTIyVDAwMTIwOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ4ODk1MDE3YTY3MjU4ZTQ2OTEwZjQyZTJiZGZmMjYyZDQ1NTYzYjg0YzhjMWViZjE3YjQ4ZjhiMWQwODUxMDUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Tl4sM0zXmvPlTiqcjg36bkbLDAtwZAyD_ufcAsfYl20">
> > Log (I cut out most of the progress, but it did stop at 47.50%):
> 
> Yours is a different problem. #447

For me it doesn't get stuck, it just stops and reports being finished. The resulted output is blank/black and nothing is saved to my drive.

**Comment by NayamAmarshe** [comment]
> > > I've got the exact same issue and can't seem to solve it, on a MBP (M1 Max, 64gb, Sonoma 14.5). <img alt="image" width="1663" src="https://private-user-images.githubusercontent.com/84971116/369626086-bac4638c-1d80-48ad-a42d-f776be977db7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY5NjQyMjgsIm5iZiI6MTcyNjk2MzkyOCwicGF0aCI6Ii84NDk3MTExNi8zNjk2MjYwODYtYmFjNDYzOGMtMWQ4MC00OGFkLWE0MmQtZjc3NmJlOTc3ZGI3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTIyVDAwMTIwOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ4ODk1MDE3YTY3MjU4ZTQ2OTEwZjQyZTJiZGZmMjYyZDQ1NTYzYjg0YzhjMWViZjE3YjQ4ZjhiMWQwODUxMDUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Tl4sM0zXmvPlTiqcjg36bkbLDAtwZAyD_ufcAsfYl20">
> > > Log (I cut out most of the progress, but it did stop at 47.50%):
> > 
> > 
> > Yours is a different problem. #447
> 
> For me it doesn't get stuck, it just stops and reports being finished. The resulted output is blank/black and nothing is saved to my drive. Also during the entire process my MB reports an excess of 20+gb of memory being free.
> 
> Should I create a separate issue for this?

No no, it's a memory issue within the NCNN backend. You're probably trying to upscale an image that is too large for the backend to handle. You should try downscaling it maybe.

**Comment by barf** [comment]
I am also struggling to get any output, it just says 'nah' during processing, reproducible on both Windows and Linux so I think Vulkan/driver issues can be ruled out.

How do we determine memory requirements or handle this type of error? It says the output was created but the file does not exist.
```
12:06:19.658 › 🎒 BACKEND REPORTED:  47.80%
12:06:19.660 › 🚧 UPSCAYL_PROGRESS:  47.80%
12:06:20.279 › 47.84%
12:06:20.280 › 🎒 BACKEND REPORTED:  47.84%
12:06:20.282 › 🚧 UPSCAYL_PROGRESS:  47.84%
12:06:20.898 › 47.88%
12:06:20.899 › 🎒 BACKEND REPORTED:  47.88%
12:06:20.901 › 🚧 UPSCAYL_PROGRESS:  47.88%
12:06:21.517 › 47.92%
12:06:21.518 › 🎒 BACKEND REPORTED:  47.92%
12:06:21.520 › 🚧 UPSCAYL_PROGRESS:  47.92%
12:06:22.137 › 47.96%
12:06:22.139 › 🎒 BACKEND REPORTED:  47.96%
12:06:22.141 › 🚧 UPSCAYL_PROGRESS:  47.96%
12:09:10.983 › 💯 Done upscaling
12:09:10.990 › 🎒 BACKEND REPORTED:  💯 Done upscaling
12:09:10.991 › upscaledImagePath:  /home/barf/Desktop/photo-bbox-2022-10k_upscayl_2x_realesrgan-x4plus.png
12:09:10.991 › 💯 UPSCAYL_DONE:  /home/barf/Desktop/photo-bbox-2022-10k_upscayl_2x_realesrgan-x4plus.png
```

**Comment by mathiasenggaard1** [comment]
Exact same issue on 2.15.0

It processes the picture from 0-78% and reports "BACKEND REPORTED:  💯 Done upscaling". but same symptoms as above. Blank preview and no saved file. 
Tried with dedicated GPU (RTX 5070) which finishes quickly, but no file. 
Tried with on-board GPU as well (Ryzen 9800 X3D) which is much. much slower but still goes from 0-100% and no file.

[Upscayl bug.txt](https://github.com/user-attachments/files/25025002/Upscayl.bug.txt)
